mxnet/inference/docker/1.5.1/py3/Dockerfile.eia

```diff
@@ -9,6 +9,7 @@ ARG MMS_VERSION=1.1.2
 ARG PYTHON=python3
 ARG PYTHON_VERSION=3.6.10
 ARG HEALTH_CHECK_VERSION=1.7.0
+ARG OPENSSL_VERSION=1.1.1k
```

Comment:
Why are we modifying MXNet containers as a part of this PR? If this is unintentional, could we revert this back?